### PR TITLE
No more use_auth_token=True

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# Checklist

- [x] Bumped `huggingface_hub` to `0.10.0`
- [x] Removed all `use_auth_token=True`.
- [x] `use_auth_token=use_auth_token` was kept since people might want to not be logged in and pass specific tokens

# TODO until release
- [ ] Update all official colabs and blog posts in the open

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ pipe = StableDiffusionPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4", 
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline README.md:117]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ pipe = StableDiffusionPipeline.from_pretrained(
     revision="fp16", 
     torch_dtype=torch.float16,
     scheduler=lms,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline README.md:143]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -169,10 +169,10 @@ pipe = StableDiffusionImg2ImgPipeline.from_pretrained(
     model_id_or_path,
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline README.md:172]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -169,10 +169,10 @@ pipe = StableDiffusionImg2ImgPipeline.from_pretrained(
     model_id_or_path,
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+    
 )
 # or download via git clone https://huggingface.co/CompVis/stable-diffusion-v1-4
-# and pass `model_id_or_path="./stable-diffusion-v1-4"` without having to use `use_auth_token=True`.
+# and pass `model_id_or_path="./stable-diffusion-v1-4"` without having to use ``.
```
Comment (patrickvonplaten) [inline README.md:175]:
```suggestion
# and pass `model_id_or_path="./stable-diffusion-v1-4"`.
```

File: README.md [hunk 5]
```diff
@@ -219,10 +219,10 @@ pipe = StableDiffusionInpaintPipeline.from_pretrained(
     model_id_or_path,
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline README.md:222]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -219,10 +219,10 @@ pipe = StableDiffusionInpaintPipeline.from_pretrained(
     model_id_or_path,
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+    
 )
 # or download via git clone https://huggingface.co/CompVis/stable-diffusion-v1-4
-# and pass `model_id_or_path="./stable-diffusion-v1-4"` without having to use `use_auth_token=True`.
+# and pass `model_id_or_path="./stable-diffusion-v1-4"` without having to use ``.
```
Comment (patrickvonplaten) [inline README.md:225]:
```suggestion
# and pass `model_id_or_path="./stable-diffusion-v1-4"`.
```

File: docs/source/optimization/fp16.mdx [hunk 7]
```diff
@@ -75,7 +75,7 @@ pipe = StableDiffusionPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4",
     revision="fp16",
     torch_dtype=torch.float16,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline docs/source/optimization/fp16.mdx:78]:
```suggestion
```

File: docs/source/optimization/fp16.mdx [hunk 8]
```diff
@@ -97,7 +97,7 @@ pipe = StableDiffusionPipeline.from_pretrained(
...
```
Comment (patrickvonplaten) [inline docs/source/optimization/fp16.mdx:100]:
```suggestion
```

File: docs/source/optimization/fp16.mdx [hunk 9]
```diff
@@ -153,7 +153,6 @@ def generate_inputs():
 pipe = StableDiffusionPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4",
     # scheduler=scheduler,
```
Comment (patrickvonplaten) [inline docs/source/optimization/fp16.mdx:155]:
```suggestion
```

File: docs/source/optimization/fp16.mdx [hunk 10]
```diff
@@ -219,7 +218,6 @@ class UNet2DConditionOutput:
...
```
Comment (patrickvonplaten) [inline docs/source/optimization/fp16.mdx:220]:
```suggestion
```

File: src/diffusers/configuration_utils.py [hunk 11]
```diff
@@ -238,7 +238,7 @@ def get_config_dict(
                     f"{pretrained_model_name_or_path} is not a local folder and is not a valid model identifier"
                     " listed on 'https://huggingface.co/models'\nIf this is a private repository, make sure to pass a"
                     " token having permission to this repo with `use_auth_token` or log in with `huggingface-cli"
-                    " login` and pass `use_auth_token=True`."
+                    " login` and pass ``."
```
Comment (patrickvonplaten) [inline src/diffusers/configuration_utils.py:241]:
```suggestion
                    " login`."
```

File: src/diffusers/modeling_flax_utils.py [hunk 12]
```diff
@@ -357,7 +357,7 @@ def from_pretrained(
                     f"{pretrained_model_name_or_path} is not a local folder and is not a valid model identifier "
                     "listed on 'https://huggingface.co/models'\nIf this is a private repository, make sure to pass a "
                     "token having permission to this repo with `use_auth_token` or log in with `huggingface-cli "
-                    "login` and pass `use_auth_token=True`."
+                    "login` and pass ``."
```
Comment (patrickvonplaten) [inline src/diffusers/modeling_flax_utils.py:360]:
```suggestion
                    "login`."
```

File: src/diffusers/modeling_utils.py [hunk 13]
```diff
@@ -338,7 +338,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                     f"{pretrained_model_name_or_path} is not a local folder and is not a valid model identifier "
                     "listed on 'https://huggingface.co/models'\nIf this is a private repository, make sure to pass a "
                     "token having permission to this repo with `use_auth_token` or log in with `huggingface-cli "
-                    "login` and pass `use_auth_token=True`."
+                    "login` and pass ``."
```
Comment (patrickvonplaten) [inline src/diffusers/modeling_utils.py:341]:
```suggestion
                    "login`."
```

File: src/diffusers/pipelines/README.md [hunk 14]
```diff
@@ -114,7 +114,7 @@ pipe = StableDiffusionImg2ImgPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4",
     revision="fp16", 
     torch_dtype=torch.float16,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/README.md:117]:
```suggestion
```

File: src/diffusers/pipelines/README.md [hunk 15]
```diff
@@ -164,7 +164,7 @@ pipe = StableDiffusionInpaintPipeline.from_pretrained(
...
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/README.md:167]:
```suggestion
```

File: src/diffusers/pipelines/stable_diffusion/README.md [hunk 16]
```diff
@@ -81,7 +81,7 @@ scheduler = DDIMScheduler(beta_start=0.00085, beta_end=0.012, beta_schedule="sca
 pipe = StableDiffusionPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4", 
     scheduler=scheduler,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/stable_diffusion/README.md:84]:
```suggestion
```

File: src/diffusers/pipelines/stable_diffusion/README.md [hunk 17]
```diff
@@ -105,7 +105,7 @@ lms = LMSDiscreteScheduler(
 pipe = StableDiffusionPipeline.from_pretrained(
     "CompVis/stable-diffusion-v1-4", 
     scheduler=lms,
-    use_auth_token=True
+
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/stable_diffusion/README.md:108]:
```suggestion
```

**Review by patil-suraj — APPROVED** [review]
Thanks a lot ! We should also remove this from the textutal_inversion and dreambooth PR

File: src/diffusers/dynamic_modules_utils.py [hunk 18]
```diff
@@ -198,7 +198,7 @@ def get_cached_module_file(
 
     <Tip>
 
-    Passing `use_auth_token=True` is required when you want to use a private model.
+    Passing `` is required when you want to use a private model.
```
Comment (pcuenca) [inline src/diffusers/dynamic_modules_utils.py:201]:
```suggestion
    You may pass a token in `use_auth_token` if you are not logged in (`huggingface-cli long`) and
    want to use private or [gated models](https://huggingface.co/docs/hub/models-gated#gated-models).
```

**Review by pcuenca — APPROVED** [review]
Awesome!

**Pull request merged**